personalitytest/static/css/personalitytest.css

```diff
@@ -1,12 +0,0 @@
-.error-span {
-  color: red;
-}
-
-.answer-cell {
-  vertical-align: middle;
-  padding-bottom: 20px;
-}
-
-.full-result-table {
-  width: 50%;
-}
```

Comment:
if you remove the CSS file you need to stop importing it [here](https://github.com/IONISx/xblock-personality-test/blob/master/personalitytest/personalitytestblock.py#L141) then.